modules/labs-react/search-form/lib/themes.ts

```diff
@@ -15,8 +15,8 @@ export interface SearchThemeAttributes {
   colorFocus?: string;
   placeholderColor?: string;
   placeholderColorFocus?: string;
-  boxShadow?: string | string[];
-  boxShadowFocus?: string | string[];
```

Comment:
This technically a breaking change, but in most cases you would just pass a string so I think it's ok. This wouldn't work with the stencil if it accepted an array i think.

Reply:
This is okay. The original code was matching the Emotion types where an array of strings meant the `boxShadow` property would be listed multiple times. This is done for browser fallbacks when a property isn't understood by the parser. It is extremely rare to do this now with our drop in IE11 support.

```jsx
boxShadow: ['first', 'second']

// CSS
box-shadow: first;
box-shadow: second;
```

Reply:
ohhh